neurodiffeq/__init__.py

```diff
@@ -23,3 +23,5 @@
 
 # Turn on future warning by default
 warnings.simplefilter('always', FutureWarning)
+
+__version__ = "0.6.3"
```

Comment:
Nit pick, let's make it `0.6.3-dev` or `0.6.3-snapshot`. If someone reports a bug, we need to know which version of neurodiffeq they are using: pypi (stable) or github (continuously changing).

Reply:
Sure.